pika/channel.py

```diff
@@ -390,7 +390,7 @@ def basic_reject(self, delivery_tag, requeue=True):
         """
         if not self.is_open:
             raise exceptions.ChannelClosed()
-        if not isinstance(delivery_tag, int):
+        if not is_integer(delivery_tag):
```

Comment:
@quantum5, I found a builtin module that may provide the same functionality in a portable way without necessitating any `pika.compat` logic:

```
import numbers

if not isinstance(delivery_tag, numbers.Intergral):
    . . .
```